x-pack/plugins/observability_solution/apm/public/components/shared/entity_enablement/welcome_modal.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiButton,
+  EuiConfirmModal,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiImage,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { useKibanaUrl } from '../../../hooks/use_kibana_url';
+
+export function Welcome({
+  isModalVisible = false,
+  onClose,
+  onConfirm,
+}: {
+  isModalVisible: boolean;
+  onClose: () => void;
+  onConfirm: () => void;
+}) {
+  const servicesInventory = useKibanaUrl('/plugins/apm/assets/services_inventory.png');
+
+  return (
+    <>
+      {isModalVisible && (
```

Comment:
Can we do it like this? I think it's cleaner!

```suggestion
if(!isModalVisible){
  return null;
}

  return (
```